examples/COUPLE/simple/simple.cpp

```diff
@@ -19,15 +19,16 @@
 //         in.lammps = LAMMPS input script
 // See README for compilation instructions
 
-#include "stdio.h"
-#include "stdlib.h"
-#include "string.h"
-#include "mpi.h"
-
-#include "lammps.h"         // these are LAMMPS include files
-#include "input.h"
-#include "atom.h"
-#include "library.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <mpi.h>
+
+// these are LAMMPS include files
+#include <lammps/lammps.h>
```

Comment:
@akohlmey: And this example is actually the reason why the code (see https://github.com/lammps/lammps/pull/627#issuecomment-325409215) used `lmp = new LAMMPS()`.